packages/components/top-bar/top-bar.ts

```diff
@@ -0,0 +1,61 @@
+import {
+    booleanAttribute,
+    ChangeDetectionStrategy,
+    Component,
+    Directive,
+    input,
+    Input,
+    ViewEncapsulation
+} from '@angular/core';
+
+@Directive({
+    standalone: true,
+    selector: '[kbqTopBarSpacer]',
+    host: {
+        class: 'kbq-top-bar-spacer'
+    }
+})
+export class KbqTopBarSpacer {}
+
+/**
+ * Directive that dynamically applying CSS classes based on a placement value (left or right).
+ */
+@Directive({
+    standalone: true,
+    selector: '[kbqTopBarContainer]',
+    host: {
+        class: 'kbq-top-bar-container',
+        '[class.kbq-top-bar-container__left]': 'placement() === "left"',
+        '[class.kbq-top-bar-container__right]': 'placement() === "right"'
+    }
+})
+export class KbqTopBarContainer {
+    /**
+     * Conditionally applies a CSS class based on the value
+     */
+    placement = input.required<'left' | 'right'>();
```

Comment:
```suggestion
    placement = input.required<'start' | 'end'>();
```
если захотим добавить поддержку RTL

Reply:
готово